pay-queue/src/pay_queue/services/eft/eft_base.py

```diff
@@ -58,10 +58,11 @@
 """This manages the EFT base class."""
 import decimal
 from datetime import datetime
+from typing import List
 
-from reconciliations.eft.eft_enums import EFTConstants
-from reconciliations.eft.eft_errors import EFTError
-from reconciliations.eft.eft_parse_error import EFTParseError
+from pay_queue.services.eft.eft_enums import EFTConstants
+from pay_queue.services.eft.eft_errors import EFTError
```

Comment:
@ochiu any issues here with some EFT changes creeping in?

Reply:
We're hoping on not sitting in this PR too long... maybe a week or two and we'll try to pull in the python branch.

Reply:
This looks fine to me. I will likely end up re-testing this when I work on generating some clean test data in the coming weeks. I can resolve any thing that comes up then.